package.json

```diff
@@ -24,27 +24,26 @@
   "homepage": "https://github.com/TehShrike/abstract-state-router",
   "dependencies": {
     "combine-arrays": "~1.0.2",
-    "eventemitter3": "2.0.3",
-    "hash-brown-router": "3.4.0",
+    "eventemitter3": "4.0.7",
```

Comment:
https://github.com/primus/eventemitter3/releases/tag/3.0.0
The differences seem pretty minor, but we do expose this interface. The exact interface wasn't described, so I'm not sure if this would be a breaking change or not.

Reply:
Oh yeah, I remember seeing that discussion in a different pull request a while back. I guess since we don't document them, it wouldn't be considered breaking?